components/mpas-ocean/src/analysis_members/Registry_high_frequency_output.xml

```diff
@@ -24,268 +24,268 @@
 		/>
-		<var name="relativeVorticityVertexAt250m" type="real" dimensions="nVertices Time" units="s^{-1}"
+		<var name="relativeVorticityVertexAt250m" type="real" dimensions="nVertices Time" units="s^-1"
 			description="relative vorticity at cell vertices at a depth of approximately 250 m"
 		/>
 		<var_array name="activeTracersAtSurface" type="real" dimensions="nCells Time">
 				<var name="temperatureAtSurface" array_group="activeTracersAtSurface" units="^\circ C" name_in_code="temperatureAtSurface"
 						description="ocean temperature at top layer"
 				/>
-				<var name="salinityAtSurface" array_group="activeTracersAtSurface" units="PSU" name_in_code="salinityAtSurface"
+				<var name="salinityAtSurface" array_group="activeTracersAtSurface" units="1.e-3" name_in_code="salinityAtSurface"
 						description="salinity at top layer"
 				/>
 		</var_array>
 		<var_array name="activeTracersAt250m" type="real" dimensions="nCells Time">
 				<var name="temperatureAt250m" array_group="activeTracersAt250m" units="^\circ C" name_in_code="temperatureAt250m"
 						description="ocean temperature at 250 m"
 				/>
-				<var name="salinityAt250m" array_group="activeTracersAt250m" units="PSU" name_in_code="salinityAt250m"
+				<var name="salinityAt250m" array_group="activeTracersAt250m" units="1.e-3" name_in_code="salinityAt250m"
 						description="salinity at 250 m"
 				/>
 		</var_array>
 		<var_array name="activeTracersAtBottom" type="real" dimensions="nCells Time">
 			<var name="temperatureAtBottom" array_group="activeTracersAtBottom" units="^\circ C" name_in_code="temperatureAtBottom"
 				description="ocean temperature at bottom"
 			/>
-			<var name="salinityAtBottom" array_group="activeTracersAtBottom" units="PSU" name_in_code="salinityAtBottom"
+			<var name="salinityAtBottom" array_group="activeTracersAtBottom" units="1.e-3" name_in_code="salinityAtBottom"
 				description="salinity at bottom"
 			/>
 		</var_array>
-			<var name="kineticEnergyAtBottom" type="real" dimensions="nCells Time" units="m^2 s^{-2}"
+			<var name="kineticEnergyAtBottom" type="real" dimensions="nCells Time" units="m^2 s^-2"
 					description="kinetic energy at cell centers at bottom"
 			/>
-			<var name="relativeVorticityAtBottom" type="real" dimensions="nCells Time" units="s^{-1}"
+			<var name="relativeVorticityAtBottom" type="real" dimensions="nCells Time" units="s^-1"
 					description="relative vorticity at cell centers at bottom"
 			/>
-			<var name="divergenceAtBottom" type="real" dimensions="nCells Time" units="s^{-1}"
+			<var name="divergenceAtBottom" type="real" dimensions="nCells Time" units="s^-1"
 					description="divergence at cell centers at bottom"
 		/>
 		<var_array name="activeTracersAvgTopto0100" type="real" dimensions="nCells Time">
 				<var name="temperatureAvgTopto0100" array_group="activeTracersAvgTopto0100" units="^\circ C" name_in_code="temperatureAvgTopto0100"
```

Comment:
same here

Reply:
and here... I'll grep for the offender to fix any others